src/components/SendButtonArrow.jsx

```diff
@@ -4,29 +4,28 @@ import NavigationArrowForward from "material-ui/svg-icons/navigation/arrow-forwa
 import PropTypes from "prop-types";
 import React, { Component } from "react";
 
-// This is because the Toolbar from material-ui seems to only apply the correct margins if the
-// immediate child is a Button or other type it recognizes. Can get rid of this if we remove material-ui
+const PADDING_AROUND_ARROW = 11;
+
 const styles = StyleSheet.create({
   container: {
-    display: "inline-block",
-    marginLeft: 20
+    position: "absolute",
+    right: -PADDING_AROUND_ARROW,
+    top: -PADDING_AROUND_ARROW,
+    padding: 0
   },
   icon: {
     color: "rgb(83, 180, 119)",
     width: 40,
     height: 40
   },
   arrowButton: {
-    "@media(minWidth: 450px)": {
+    "@media(min-width: 450px)": {
       display: "none !important"
     },
-    "@media(maxWidth: 450px)": {
+    "@media(max-width: 450px)": {
       display: "block !important"
     },
```

Comment:
If the send arrow should be visible on larger screen sizes (as it was before this change), I can remove this for the following result.

![after-wo-media](https://user-images.githubusercontent.com/8495791/104956233-b666f080-5999-11eb-96ee-7e13ed10c900.png)